pom.xml

```diff
@@ -696,6 +696,47 @@
                         </execution>
                     </executions>
                 </plugin>
+                <plugin>
+                    <groupId>com.diffplug.spotless</groupId>
+                    <artifactId>spotless-maven-plugin</artifactId>
+                    <version>2.44.4</version>
+                    <configuration>
+                        <formats>
+                            <format>
+                                <trimTrailingWhitespace/>
+                                <endWithNewline/>
+                                <indent>
+                                    <spaces>true</spaces>
+                                    <spacesPerTab>2</spacesPerTab>
+                                </indent>
+                                <includes>
+                                    <include>.gitignore</include>
+                                </includes>
+                                <excludes>
+                                    <exclude>**gradlew**</exclude>
+                                    <exclude>**mvnw**</exclude>
+                                </excludes>
+                            </format>
+                        </formats>
+                        <java>
+<!--                            <googleJavaFormat/>-->
+                            <palantirJavaFormat/>
```

Comment:
considering the changes done they seem okayish. Its not changing string equals like expected from PMD. this is an delta needs to be investigated.



<img width="1885" alt="image" src="https://github.com/user-attachments/assets/40282026-2364-4097-8c66-bf0f5cd8de08" />

Reply:
pmd error seems to be depending on whitespace this is strange: 

format should not break PMD as not changing logic just format:


<img width="1614" alt="image" src="https://github.com/user-attachments/assets/2adde84c-e168-450b-8b39-08ac4d9084e7" />



127:

<img width="1913" alt="image" src="https://github.com/user-attachments/assets/7f7388b3-d7d6-4fb8-ae9b-dd9c5b3f4bf3" />

Reply:
ah yes, thank you.